modules/test/playwright/tests/frontend-data-set-views-web/creationActions.spec.ts

```diff
@@ -618,6 +625,53 @@ fragmentTest.describe('Creation Actions in the fragment', () => {
 
 				await expect(dialog).not.toBeInViewport();
 			});
+
+			await fragmentTest.step(
+				'Creation Action of type "modal" with custom header (Commerce) does not duplicate the title',
+				async () => {
+					await commerceAdminProductPage.goto();
+
+					await commerceAdminProductPage.page
+						.getByRole('heading', {name: 'Products'})
+						.waitFor({state: 'visible'});
```

Comment:
We can now remove all tests on commerce pages, right? Do we need this test on both creation actions and item actions?

Reply:
:facepalm: I thought that those were removed